components/InputBox.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export interface InputBoxProps {
+  // eslint-disable-next-line prettier/prettier
+
+  /** Type of the input. */
+  inputType: string;
+
+  /** Name of the input. */
+  inputName: string;
+
+  /** Placeholder of the input. */
+  placeholder: string;
+
+  /** Value of the input. */
+  inputValue: string;
+
+  /** The function to set value of the input. */
+  setInput: (value: string) => void;
+}
```

Comment:
Add this interface in `component/types` folder and export it from there as it will be used in other components as well.

Reply:
Done